mdk/db.py

```diff
@@ -69,6 +69,22 @@ def __init__(self, engine, options):
             except:
                 raise Exception('Connexion failed! Make sure the database \'%s\' exists.' % str(options['user']))
 
+        elif engine == 'sqlsrv':
+            # pyodbc.
+            host = str(options['host'])
+            port = int(options['port'])
+            user = str(options['user'])
+            password = str(options['passwd'])
+            connectionstr = "DRIVER={ODBC Driver 13 for SQL Server};SERVER=%s;PORT=%d;UID=%s;PWD=%s" \
+                            % (host, port, user, password)
+            self.conn = pyodbc.connect(connectionstr)
+            self.conn.autocommit = True
+
+            try:
+                self.cur = self.conn.cursor()
+            except:
+                raise Exception('Connexion failed! Make sure the database \'%s\' exists.' % str(options['user']))
```

Comment:
Is this error message relevant, or a copy/paste from Postegres left behind? With postgres, the connection always fails if a database of the same name of the user does not also exist.